ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -157,6 +176,85 @@ export const ProfilePage = withUserProfile()(class extends React.Component<
       </div>;
     };
 
+    function renderInstitutionalAffiliationComponents(componentHandle: React.Component) {
```

Comment:
I would definitely annotate each of these functions with the return type to enforce that they need to return some sort of React.Fragment as part of their contract.